core/store/src/main/java/uk/gov/gchq/gaffer/store/Store.java

```diff
@@ -1096,7 +1098,8 @@ private void addCoreOpHandlers() {
         addOperationHandler(GetTraits.class, getGetTraitsHandler());
 
         //Imports
-        addOperationHandler(ImportCsv.class, new ImportCsvHandler());
+        addOperationHandler(CsvToElements.class, new CsvToElementsHandler());
+        addOperationHandler(ImportFromLocalFile.class, new ImportFromLocalFileHandler());
```

Comment:
I don't think these should be here at all. These shouldn't be core operations, the user/admin should need to have actively enabled them before they can be used.